addons/account/wizard/account_move_reversal_view.xml

```diff
@@ -41,7 +41,7 @@
                     </group>
                     <footer>
                         <button string='Reverse' name="reverse_moves" type="object" class="btn-primary" data-hotkey="q"/>
-                        <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="z"/>
+                        <button string="Discard" class="btn-secondary" special="cancel" data-hotkey="z"/>
```

Comment:
Hello @odoo/rd-accounting,

Do you agree about that change? It is to be consistent with the default button since always edit feature. Now, the cancel button has `Discard` as label. 🙂 

If not, we can remove that changes. 🙂

Reply:
@william-andre Discard is the proper term for an action. Those buttons are actions, since they are in wizard. LGTM.